sw_i2c/sample-implementations/Atmel_SAMD2_series/sensirion_sw_i2c_implementation.c

```diff
@@ -101,7 +110,13 @@ u8 sensirion_SCL_read() {
 
 /**
  * Sleep for a given number of microseconds. The function should delay the
- * execution for at least the given time, but may also sleep longer.
+ * execution approximately the given time.
```

Comment:
I think it's important to leave the "at least the given time", since a longer delay doesn't cause problems, just slower communication, but a shorter one can lead to problems.

Reply:
It's not just used for coms, but currently there's no problems with longer delays either in those cases. I'll re-add the lower bound.